app/scripts/modules/amazon/src/loadBalancer/configure/classic/advancedSettings.html

```diff
@@ -3,16 +3,24 @@
 
     <div class="form-group">
       <div class="col-md-6 sm-label-right">
-        <b>Timeout</b><help-field key="ctrl.loadBalancerCommand.advancedSettings.healthTimeout"/>
+        <b>Timeout</b><help-field key="loadBalancer.advancedSettings.healthTimeout"/>
       </div>
       <div class="col-md-4">
-        <input class="form-control input-sm" type="number" min="0" ng-model="ctrl.loadBalancerCommand.healthTimeout"/>
+        <input class="form-control input-sm"
+               type="number"
+               min="0"
+               name="healthTimeout"
+               ng-model="ctrl.loadBalancerCommand.healthTimeout"
+               ui-validate="{ timeout: 'ctrl.validateTimeout($value)' }"/>
```

Comment:
Oh.....clever.  I was trying to think of a way to not have to pull in something.  Lemme test that out.